acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PassportAPIPage.java

```diff
@@ -0,0 +1,283 @@
+package uk.gov.di.ipv.cri.passport.acceptance_tests.pages;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.SignedJWT;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.di.ipv.cri.passport.acceptance_tests.model.ClientResponse;
+import uk.gov.di.ipv.cri.passport.acceptance_tests.model.DocumentCheckResponse;
+import uk.gov.di.ipv.cri.passport.acceptance_tests.service.ConfigurationService;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.text.ParseException;
+import java.util.Base64;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class PassportAPIPage extends PassportPageObject {
+
+    private static String SESSION_REQUEST_BODY;
+    private static String SESSION_ID;
+    private static String STATE;
+    private static String AUTHCODE;
+    private static String ACCESS_TOKEN;
+    private static String VC;
+    private static String RETRY;
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private final ConfigurationService configurationService =
+            new ConfigurationService(System.getenv("ENVIRONMENT"));
+    private static final Logger LOGGER = Logger.getLogger(PassportAPIPage.class.getName());
+
+    public String getAuthorisationJwtFromStub(String criId, Integer LindaDuffExperianRowNumber)
```

Comment:
I think the common aspects of this should be updated to align with DL elements of this are for old passport